mars/services/web/ui/src/task_info/TaskListPage.js

```diff
@@ -32,18 +32,36 @@ class TaskList extends React.Component {
         this.state = {};
     }
 
+    fetchTilebaleGraph() {
```

Comment:
Is this function added only for test purpose? If so, add it later when we actually implements task visualization frontend.